jobs/build-arch.Jenkinsfile

```diff
@@ -234,8 +227,6 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}") {
                     /run/kubernetes/secrets/fedora-messaging-coreos-key
                 cosa remote-session sync {,:}/run/kubernetes/secrets/fedora-messaging-coreos-key/
             fi
-            # sync over the send-ostree-import-request.py from the automation repo
-            cosa remote-session sync {,:}/var/tmp/fcos-releng/coreos-ostree-importer/send-ostree-import-request.py
```

Comment:
@dustymabe Is there a reason we were running the script from the remote instead of directly on the controller? I couldn't think of one so simplified this, but I can undo it if required.

Reply:
From memory (may lack accuracy): I think it's because the script does some verification at the end that required some of the local files. I know that's true for the signing requests we send.. Maybe not for the OSTree import request.